.github/actions/set-dev-env/action.yml

```diff
@@ -9,7 +9,11 @@ inputs:
   pandoc:
     description: Whether this should install pandoc or not
     required: false
-    default: "False"
+    default: "false"
+  graphviz:
+    description: Whether this should install Graphviz or not
+    required: false
+    default: "false"
```

Comment:
I tried to change this to the GH actions boolean `false` and setting the `type: boolean` here but GH actions complained that this had to be a `string` 🤷🏽  not sure why so left it as a string for now